packages/core/src/connectors/browser.ts

```diff
@@ -0,0 +1,156 @@
+import { AccountInterface } from 'starknet'
+import { Connector } from './base'
+import { IStarknetWindowObject } from 'get-starknet'
+import {
+  ConnectorNotConnectedError,
+  ConnectorNotFoundError,
+  UserNotConnectedError,
+  UserRejectedRequestError,
+} from '../errors'
+
+export interface BrowserConnectorOptions {
+  id: string
+}
+
+export class BrowserConnector extends Connector<BrowserConnectorOptions> {
```

Comment:
Is the concern around graceful deprecation? I think that is a good alternative for now. I also wouldn't be opposed to the breaking change tbh, since we are so early and it is quite easy to migrate

Reply:
Ok, I changed it back to `InjectedConnector`.